convex/invitations.ts

```
@@ -0,0 +1,482 @@
+      });
+
+      return {
+        email: invitation.email,
+        fullName: invitation.fullName,
+        role: invitation.role,
+        profileId,
+      };
+    } catch (error) {
+      // If profile creation fails, we don't revert the token usage
+      // This prevents the same token from being tried again
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)